dict/portuguese.yml

```diff
@@ -0,0 +1,291 @@
+    'UZ': [Uzbequistão, Usbequistão, República do Uzbequistão, Usbeque*, Uzbeque*, Usbequistanês*, Tasquente, Tashkent]
+
+  EAST:
+    'CN': [China, República Popular da China, RPC, Chines*, Chinês*, Pequim, Xangai, Shanghai]
+    'HK': [Hong Kong, Honconguês, Honcongues*]
+    'JP': [Japão, Estado do Japão, Japonês*, Japones*, Japonense*, Japônic*, Japónic*, Nipônic*, Nipónic*, Tóquio, Metrópole de Tóquio]
+    'KP': [Coreia do NORTH, Coréia do NORTH, República Popular Democrática da Coreia, RPDC, NORTH-Corean*, Pyongyang, Pionguiangue]
+    'KR': [Coreia do SOUTH, Coréia do SOUTH, República da Coreia, Coreia, SOUTH-Corean*, Seul, Cidade Especial de Seul]
+    'MN': [Mongólia, Mongol*, Mongólic*, Ulã Bator, Ulan Bator]
+    'MO': [Macau, Macauense*, Macaense*, Macaíst*]
+    'TW': [Taiwan, Taiwanês*, Formosan*, Formosin*, Taiuanês*, Taipé, Cidade de Taipé]
+
+  SOUTH:
+    'AF': [Afeganistão, Emirado Islâmico do Afeganistão, Afegã*, Afegan*, Afegânic*, Cabul, Cábul]
+    'BD': [Bangladesh, Bangladexe, Bangladeche, Bangladês*, Banglades*, Bangladense*, Bengal*, Bangladexense*, Bangladexian*, Bangladechian*, Bangladeshian*, Bengalês, Bengalense*, Bengala*, Bengali*, Daca, Dhaka]
+    'BT': [Butão, Reino do Butão, Butanês*, Butanense*, Butani*, Butâni*, Timbu, Thimphu, Timpu, Thimbu]
+    'IN': [Índia, República da Índia, Indian*, Bombaim, Mumbai, Nova Deli, Nova Delhi, Nova Déli]
+    'IR': [Irã, Irão, República Islâmica do Irã, República Islâmica do Irão, Iranian*, Teerã, Teerão]
+    'LK': [Sri Lanka, Sri Lanca, Seri Lanca, República Democrática Socialista do Sri Lanka, Ceilão, Cingalês*, Srilankês*, Srilanquês*, Colombo, Columbo]
+    'MV': [Maldivas, República das Maldivas, Maldiv*, Maldívi*]
+    'NP': [Nepal, República Democrática Federal do Nepal, Nepalês, Nepales*, Catmandu]
+    'PK': [Paquistão, República Islâmica do Paquistão, Paquistanês*, Islamabade, Islamabad]
+
+  SOUTH EAST:
+    'BN': [Brunei, Estado de Brunei Darussalam, Estado de Brunei Darussalã, Brunean*, Bruneín*]
+    'ID': [Indonésia, República da Indonésia, Indonési*, Jacarta]
+    'KH': [Camboja, Cambodja, Reino do Camboja, Cambojan*, Phnom Penh, Pnom Pen, Pnom Pene, Penome Pene]
+    'LA': [Laos, República Democrática Popular do Laos, Laosian*, Laocian*, Vientiane, Vienciana]
+    'MM': [Myanmar, Birmânia, República da União de Myanmar, Mianmarense*, Birmanês*, Birmã*, Birman*, Birmanian*, Birmanense*, Rangum, Yangon, Nepiedó, Naypyidaw, Nay Pyi Taw]
+    'MY': [Malásia, Malaio*, Malaia*, Malási*, Kuala Lumpur, Cuala Lumpur, Quala Lumpur, Cualalampur, Putrajaia, Putrajaya]
+    'PH': [Filipinas, República das Filipinas, Filipin*,  Manila]
+    'SG': [Singapura, Cingapura, Singapurense*, Singapuran*]
+    'TH': [Tailândia, Reino da Tailândia, Tailandês*, Banguecoque, Bangkok, Bancoque]
+    'TL': [Timor-EAST, República Democrática de Timor-EAST, Timorense*, Maubere*, Díli]
```

Comment:
Isn't it "East Timor" (according to Google Translate)?

Reply:
Same problem as the two previous cases.